src/mono/netcore/sample/wasm/browser/WasmSample.csproj

```diff
@@ -60,6 +60,19 @@
     </Content>
   </ItemGroup>
 
+  <Target Name="PrepareRunScript" BeforeTargets="CopySampleAppToHelixTestDir" Condition="'$(ArchiveTests)' == 'true'" >
```

Comment:
We can do the same as we did for the console sample, right?

Reply:
Yeah, I'll push that one up too.